src/sentry/static/sentry/app/views/usageStats/utils.tsx

```diff
@@ -0,0 +1,87 @@
+import {DataCategory} from 'app/types';
+
+export const MILLION = 10 ** 6;
+export const BILLION = 10 ** 9;
+export const UNLIMITED = '∞';
+
+export const GIGABYTE = 10 ** 9;
+
+type formatOptions = {
+  isAbbreviated?: boolean;
+  useUnitScaling?: boolean;
+};
+
+/**
+ * This expects values from CustomerUsageEndpoint, which contains usage
```

Comment:
I don't think `CustomerUageEndpoint` is in sentry.

Reply:
It isn't.

Reply:
Lol it isn't for sure. I just copied the method + comments over without looking.

Reply:
Lol sorry